setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import find_packages, setup
```

Comment:
do we really need setup.py?
If we use poetry, it can handle building packages without setup.py (if I am not mistaken)